src/guide/command-line-options.md

```diff
@@ -107,6 +107,38 @@ vendor/bin/phpunit --coverage-xml=build/coverage/coverage-xml --log-junit=build/
 infection.phar --coverage=build/coverage
 ```
 
+### `--git-diff-filter`
+
+Allows filtering files to mutate by using `git diff` with `--diff-filter` option. Sensible values are: `AM` - added and modified files. `A` - only added files.
+
+Can be used on CI, e.g. with GitHub Actions, TravisCI and so on.
+
+Usage:
+
+```bash
+infection.phar --git-diff-filter=A
+```
+
+This command will mutate only those files, that were added in the Pull Request. The diff is done between the current branch and `origin/master`.
```

Comment:
```suggestion
This command will mutate only those files, that were added in the Pull Request. The diff is done between the current branch and the base branch.
```
New repos on GH use `main` branch by default, [so will the Git itself](https://github.com/git/git/commit/a15ad5d1bc23f3a6842360df5ee840b1dc3e4888).